TM1py/Services/MonitoringService.py

```diff
@@ -27,6 +27,16 @@ def get_threads(self, **kwargs) -> List:
         response = self._rest.GET(url, **kwargs)
         return response.json()['value']
 
+    def get_active_threads(self, **kwargs):
+        '''Return a list of non-idle threads from the TM1 Server
```

Comment:
Triple double-quotes should be used for docstrings
https://www.python.org/dev/peps/pep-0257/